docs/reference/cluster/reroute.asciidoc

```diff
@@ -58,28 +58,30 @@ query parameter, which will attempt a single retry round for these shards.
 ==== {api-query-parms-title}
 
 `dry_run`::
-    (Optional, boolean) If `true`, then the request simulates the operation only 
+    (Optional, boolean) If `true`, then the request simulates the operation only
     and returns the resulting state.
-    
+
 `explain`::
-    (Optional, boolean) If `true`, then the response contains an explanation of 
+    (Optional, boolean) If `true`, then the response contains an explanation of
     why the commands can or cannot be executed.
 
 `metric`::
-    (Optional, string) Limits the information returned to the specified metrics. 
+    (Optional, string) Limits the information returned to the specified metrics.
     Defaults to all but metadata The following options are available:
 
 +
---
+.Properties of `metric`
+[%collapsible%open]
+======
```

Comment:
```suggestion
Because these are query parameters, they're not objects _per se_. It's more accurate to describe them as `options.

I'm not sure we need the collapsible section here as these won't be nested more than one level deep.

.Options for `metric`
[%collapsible%open]
======
```